src/collectors/file_collector_imp.hpp

```diff
@@ -0,0 +1,103 @@
+#ifndef BAYESMIX_COLLECTORS_FILE_COLLECTOR_IMP_HPP_
+#define BAYESMIX_COLLECTORS_FILE_COLLECTOR_IMP_HPP_
```

Comment:
Is "_imp.hpp" the correct suffix, as in Google styleguide-compliant, for this kind of file?

Reply:
right: I moved all the implementations in the file_collector.hpp file